packages/@aws-cdk/aws-appmesh/test/integ.mesh.expected.json

```diff
@@ -95,15 +95,15 @@
     "vpcPublicSubnet1NATGateway9C16659E": {
       "Type": "AWS::EC2::NatGateway",
       "Properties": {
+        "SubnetId": {
```

Comment:
This seems unrelated to the PR. Can we revert this?

Reply:
This actually looks like just switching the position for `SubnetId` and `AllocationId`.

Please be noted `integ.mesh.expcted.json` file is generated automatically after running `yarn integ integ.mesh.js` command.